scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/data/TaskProActiveDataspaces.java

```diff
@@ -375,12 +378,12 @@ private void logDataspacesStatus(String message, DataspacesStatusLevel level) {
         final String eol = System.lineSeparator();
         final boolean hasEol = message.endsWith(eol);
 
-        if (level == DataspacesStatusLevel.ERROR) {
-            this.clientLogs.append("[DATASPACES-ERROR] ").append(message).append(hasEol ? "" : eol);
-        } else if (level == DataspacesStatusLevel.WARNING) {
-            this.clientLogs.append("[DATASPACES-WARNING] ").append(message).append(hasEol ? "" : eol);
-        } else if (level == DataspacesStatusLevel.INFO) {
-            this.clientLogs.append("[DATASPACES-INFO] ").append(message).append(hasEol ? "" : eol);
+        if (taskLogger != null && level == DataspacesStatusLevel.ERROR) {
+            taskLogger.getErrorSink().println("[DATASPACES-ERROR] " + message + (hasEol ? "" : eol));
```

Comment:
println() comes with an EOL characters (= CR/LF/CRLF). Therefor, I bet you wanted to use print(), and not println()

Reply:
you are right